python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1326,6 +1334,11 @@ def __get_all_vertices_series(self):
         if epd is not None:
             vert_sers.append(epd[self.src_col_name])
             vert_sers.append(epd[self.dst_col_name])
+        if len(vert_sers) > 1 and not all(
+            cudf.api.types.is_dtype_equal(vert_sers[0].index.dtype, s.index.dtype)
+            for s in vert_sers
+        ):
+            vert_sers = [s.reset_index(drop=True) for s in vert_sers]
```

Comment:
I believe these lines aren't needed for SG, and I removed them from https://github.com/rapidsai/cugraph/pull/2804.  Can you remove it and verify things still work for you?

Reply:
> 

I removed them and confirmed the tests still pass.

Reply:
The FIXMEs are for future PRs.  They're mostly referring to planned features that are not urgent.

Reply:
And I removed the print statements.